boolean/condition/a9ccc258-8320-46cd-a29a-180316401e63.json

```diff
@@ -1,22 +1,22 @@
 {
-  "description": "",
+  "description": "Outputs either on the \"true\" or the \"false\" output port, depending on the input value.",
```

Comment:
When referenceing ports, backticks should be used instead of quotes to keep consistency w/ types.

Reply:
fixed